src/test/resources/uk.gov.homeoffice.digital.sas.timecard.bdd/timecard.feature

```diff
@@ -15,8 +15,121 @@ Feature: Timecard
     When Trevor creates the valid time-entries in the timecard service
     Then the last response should have a status code of 200
     Then the 1st of the time-entries in the last response should contain
-      | field           | type    | expectation                                       |
-      | id              | String  | isNotNull()                                       |
-      | ownerId         | String  | isEqualTo("00000000-0000-0000-0000-000000000001") |
-      | actualStartTime | Instant | isEqualTo("2022-11-16T08:00:00Z")                 |
-      | actualEndTime   | Instant | isNull()                                          |
+      | field            | type    | expectation                                                |
+      | id               | String  | isNotNull()                                                |
+      | ownerId          | String  | isEqualTo(personaManager.getPersona('Trevor').id.toString) |
+      | actualStartTime  | Instant | isEqualTo("2022-11-16T08:00:00Z")                          |
+      | actualEndTime    | Instant | isNull()                                                   |
+      | timePeriodTypeId | String  | isEqualTo(resourceHelper.getResourceId('Trevor', 'timecard','time-period-types','name=="Shift"').toString) |
+
+
+  Scenario: Create time entry with end date equal to the start date
+
+    Given Trevor is a user
+    And the invalid time-entries are
+      """
+      {
+        "ownerId": "#{personaManager.getPersona('Trevor').id.toString}",
+        "timePeriodTypeId": "#{resourceHelper.getResourceId('Trevor', 'timecard','time-period-types','name=="Shift"')}",
+        "actualStartTime": "2022-11-16T10:00:00Z",
+        "actualEndTime": "2022-11-16T10:00:00Z"
+      }
+      """
+    When Trevor creates the invalid time-entries in the timecard service
+    Then the last response should have a status code of 200
```

Comment:
Shouldn't this return a 400 and an error message

Reply:
I realise I updated the wording of this, when with the current rules, this is valid. If this behaviour is incorrect, would fixing it come under this ticket, or a separate bugfix?

Reply:
Bug ticket raised here: https://collaboration.homeoffice.gov.uk/jira/browse/EAHW-2640